src/ell_set.cpp

```diff
@@ -156,7 +157,14 @@ static int ellps_ellps (PJ *P) {
     err = proj_errno_reset (P);
 
     paralist* new_params = pj_mkparam (ellps->major);
+    if (nullptr == new_params)
+        return proj_errno_set (P, ENOMEM);
     new_params->next = pj_mkparam (ellps->ell);
+    if (nullptr == new_params)
```

Comment:
cppcheck complains:
```
/home/travis/build/OSGeo/PROJ/src/ell_set.cpp:163,warning,nullPointer,Possible null pointer dereference: new_params - otherwise it is redundant to check it against null.
```
Indeed the check should be against new_params->next